gossip/src/cluster_info.rs

```diff
@@ -2749,47 +2496,50 @@ impl Node {
             num_tvu_receive_sockets.get(),
         )
         .expect("tvu multi_bind");
-
         let (tvu_quic_port, tvu_quic) =
-            Self::bind_with_config(bind_ip_addr, port_range, socket_config);
+            bind_in_range_with_config(bind_ip_addr, port_range, socket_config)
+                .expect("tvu_quic bind");
 
-        let (tpu_port, tpu_sockets) =
-            multi_bind_in_range_with_config(bind_ip_addr, port_range, socket_config, 32)
-                .expect("tpu multi_bind");
-
-        let (_tpu_port_quic, tpu_quic) = Self::bind_with_config(
-            bind_ip_addr,
-            (tpu_port + QUIC_PORT_OFFSET, tpu_port + QUIC_PORT_OFFSET + 1),
-            socket_config,
-        );
-        let tpu_quic =
-            bind_more_with_config(tpu_quic, num_quic_endpoints.get(), socket_config).unwrap();
+        let ((tpu_port, tpu_sockets), (_tpu_port_quic, tpu_quic)) =
+            bind_two_in_range_with_offset_and_config(
+                bind_ip_addr,
+                port_range,
+                QUIC_PORT_OFFSET,
+                socket_config,
+                socket_config,
+            )
+            .expect("tou_socket primary bind");
+        let tpu_sockets =
+            bind_more_with_config(tpu_sockets, 32, socket_config).expect("tpu_sockets multi_bind");
```

Comment:
Why hard coded the count? Should be based on the request

Reply:
it has been hardcoded before. Also, do we even use use a single one of those 32? I thought TPU UDP is not used anymore.

Reply:
@sakridge confirmed it is going away so we do not care at the moment.